lib/tasks/create_organisations.rake

```diff
@@ -11,5 +11,11 @@ namespace :data do
         Organisation.create(school_group.attributes)
       end
     end
+
+    desc 'Update organisations data'
```

Comment:
indentation looks slightly out here?

Reply:
Indentation is correct here, it has to be aligned to the task